CHANGELOG.md

```diff
@@ -97,7 +102,16 @@ of response and not the raw API endpoint response
   - **CUMULUS-2634**
     - Changed `sfEventSqsToDbRecords` Lambda to use new upsert helpers for executions, granules, and PDRs
     to ensure out-of-order writes are handled correctly when writing to Elasticsearch
-    
+  - **CUMULUS-2510**
+    - Removed `stream_enabled` and `stream_view_type` from `collections_table`
+      TF definition.
+    - Removed `aws_lambda_event_source_mapping` TF definition on collections
+      DynamoDB table.
+    - Updated `@cumulus/api/endpoints/collections` to publish an SNS message
+      after a successful collection delete, update (PUT), create (POST).
+    - Updated functions `create` and `upsert` in the `db` model for Collections
+      to return an array of objects containing values of the created or updated
+      columns.
```

Comment:
```suggestion
      to return an array of objects containing all columns for the created or updated records
```

I believe `returning` returns whatever columns you specify whether they were changed or not:

https://knexjs.org/#Builder-returning

So `*` would always return all columns

Reply:
Ah, I think that was what I was trying to articulate but your suggestion is much better [`754f871` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/754f871e6b2a2231b24c1dc75a24f09bf623fabd)